apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.js

```diff
@@ -0,0 +1,78 @@
+const error = require('@prairielearn/error');
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const router = express.Router();
+const { loadSqlEquiv } = require('@prairielearn/postgres');
+const sqldb = require('@prairielearn/postgres');
+const sql = loadSqlEquiv(__filename);
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const params = {
+      assessment_id: res.locals.assessment.id,
+      current_student_uid: res.locals.user.uid,
+      current_assessment_title: res.locals.assessment.title,
+    };
+    const result = await sqldb.queryAsync(sql.select_assessment_access_policy, params);
+    res.locals.policies = result.rows;
+    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+  }),
+);
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    if (req.body.__action === 'add_new_override') {
+      const params = {
+        assessment_id: res.locals.assessment.id,
+        created_by: res.locals.user.uid,
```

Comment:
Was this tested? `created_by` is a `BIGINT` column, but `res.locals.user.uid` is a string.